controllers/sriovoperatorconfig_controller.go

```diff
@@ -120,6 +122,12 @@ func (r *SriovOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.
 		return reconcile.Result{}, err
 	}
 
+	// For Openshift we need to create the systemd files using a machine config
+	if utils.ClusterType == utils.ClusterTypeOpenshift && defaultConfig.Spec.ConfigurationMode == sriovnetworkv1.SystemdConfigurationMode {
```

Comment:
Need to handle hypershift too

Reply:
this is how we check hypershift (dn.openshiftContext.IsHypershift()) like here
```
 utils.ClusterType == utils.ClusterTypeOpenshift && !dn.openshiftContext.IsHypershift()
```

Reply:
I mean that in the body of the conditional that checks for openshift, we need to handle hypershift differently than normal openshift.

Reply:
what will be the difference in hypershift?
there is no machine config?